src/generate/index.js

```diff
@@ -2,7 +2,7 @@ const _ = require('lodash/fp')
 const formatBadge = require('./format-badge')
 const formatContributor = require('./format-contributor')
 
-const badgeRegex = /\[!\[All Contributors\]\([a-zA-Z0-9\-./_:?=]+\)\]\(#\w+\)/
+const badgeRegex = /\[!\[All Contributors\]\([a-zA-Z0-9\-./_:?=]+\)\]\(#[\w\-]+\)/
```

Comment:
> [lint] /home/circleci/repo/src/generate/index.js
> [lint]   5:76  error  Unnecessary escape character: \-  no-useless-escape

Reply:
Oh! I forgot to execute Lint... :woman_facepalming: 
Thanks @Berkmann18 , done!!

Reply:
@Berkmann18 , please, can I ask you a question? Why did Lint find only one of them? I mean, `-` is escaped before. It's only a curiosity. Thanks again for your help!!

Reply:
@Berkmann18 , it seems it only happens for `-`as the first element or the last element in a RE set. If it's the first or the last element, it's not indicating a range so it's not necessary. Thanks again!!

Reply:
@rachelcarmena Normally the pre-commit hook should automatically action ESLint (depending on the file extension) so you either used `--no-verify` or it bugged somewhere.
> please, can I ask you a question? Why did EsLint find only one of them? I mean, - is escaped before. It's only a curiosity. Thanks again for your help!!

Good catch, I guess it only looked at the one changed but that wouldn't really make sense considering it's on the same line.

> it seems it only happens for -as the first element or the last element in a RE set. If it's the first or the last element, it's not indicating a range so it's not necessary. Thanks again!!

After reflecting on that, RE's grammar is such that `-` placed between characters (e.g: `[a-z]`, `[0-9]`) acts as the range operator (so /[a-z]/ would match any single lower case letters and /[0-9]/, any single digits). Whereas escaping it (`\-`) would tell the RE parser that it's an **element of the set**.

In other words: /[a-c]/ accepts 'a', 'b' and 'c' whereas /[a\-c]/ only accepts 'a', '-' and 'c'.

I hope that makes sense.

Reply:
Thanks @Berkmann18 ! It makes sense.